presenter/src/main/java/com/mashup/chinchin/presenter/add_friend/AddFriendActivity.kt

```diff
@@ -23,27 +26,36 @@ import dagger.hilt.android.AndroidEntryPoint
 class AddFriendActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        // FIXME: ViewModel 추가 되면 이관
+        val newFriend = intent.extras?.get(NEW_FRIEND) as? FriendUiModel // TODO: friendProfileUiModel?로 바꿔야하네..
+
         setContent {
             ChinchinTheme {
                 AddFriendScreen(
                     onActivityFinish = { finish() },
+                    newFriend = newFriend,
                 )
             }
         }
     }
+
+    companion object {
+        const val NEW_FRIEND = "NEW_FRIEND"
+    }
 }
 
 @Preview(showBackground = true)
 @Composable
 fun AddFriendPreview() {
-    AddFriendScreen()
+    AddFriendScreen(newFriend = null)
 }
 
 @Composable
 fun AddFriendScreen(
     onActivityFinish: () -> Unit = {},
+    newFriend: FriendUiModel?,
```

Comment:
이거는 default argument를 지정해주는게 좋아보여
```suggestion
    newFriend: FriendUiModel? = null,
```

새로운 친구를 추가하지 않을 케이스에도 무조건적으로 null값을 보내지 않을 수 있는 상황을 만들어주는게 좋을거같아

Reply:
[3bbd0b3](https://github.com/mash-up-kr/chinchin-android/pull/93/commits/3bbd0b375099708f3d81282852424af33145799f) 
수정했습니다.